packages/loopring_v3/contracts/core/impl/ExchangeV3.sol

```diff
@@ -111,6 +112,15 @@ contract ExchangeV3 is IExchangeV3, ReentrancyGuard
         state.agentRegistry = IAgentRegistry(_agentRegistry);
     }
 
+    function refreshBlockVerifier()
+        external
+        nonReentrant
+        onlyOwner
+    {
+        require(state.loopring.blockVerifierAddress() == address(0), "ZERO_ADDRESS");
```

Comment:
`require(state.loopring.blockVerifierAddress() != address(0), "ZERO_ADDRESS");`?

Reply:
updated.

Reply:
Should be `require(state.loopring.blockVerifierAddress() != address(0), "ZERO_ADDRESS");`, `require(state.loopring.blockVerifierAddress() !== address(0), "ZERO_ADDRESS");` it too much javascript. :)

Reply:
Sorry.